doc/common/README.md

```diff
@@ -569,14 +569,51 @@ d. Участники, проверяющие код, добавляют ком
 
 e. Комментарии отмечает выполненными автор изменений, код которого просматирвают.
 
-f. После корректировки замечаний и реализации предложений должен запрашиваться
+f. В начале процесса исправления замечаний задача переводится в соотвествующий рабочий статус. 
+   После завершения исправлений задача переводится обратно в статус ревью.
+
+g. После корректировки замечаний и реализации предложений должен запрашиваться
 повторный обзор кода.
 
-g. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
+h. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
 использовать такой способ коммуникации, при котором ответ затягивается, а соответственно
 затягивается и ревью.
 
-#### 6.4.4. Противоречия и споры в процессе обзора кода
+#### 6.4.4. Процесс обзора задачи, не связанных с системой управления кодом (Github, Gitlab)
+
+Такие задачи, как документирование в Notion, Confluence и пр. 
+
+Мотивация:\
+Сократить количество отвлекающих воздействий на автора изменений до 2 (в позитивном сценарии), 
+меньше отвлекая друг друга от работы. 
+
+1. Рецензент в процессе обзора кода фиксирует свои замечания и пожелания в одном 
+   месте. \
+   Например: тематический канал в Slack, с отключёнными уведомлениями. \
```

Comment:
```suggestion
   Например: канал `review` в Slack, где уже есть ссылка на задачу от автора изменений. \
   Соответственно он получит уведомление об оставленном комментарии.
```
It is a little bit contradict with origin idea, but there is no problem if we activate special mode that described in comment above.

The bonuses:
* there is no additional channel
* we guard ourselves not only from review messages, but also from any other
* we can use telegram for some exceptional cases (it should be very rare case)